index.bs

```diff
@@ -3280,7 +3286,7 @@ and at least one of the following three conditions is true:
                     <span>callback function</span>
             </div></th>
                 <th><div>
-                    <span>dictionary</span>
+                    <span>[open] dictionary</span>
```

Comment:
So others might have different opinions here, but overall, I feel like this section is rather complicated and I'm not sure this actually makes it more readable.